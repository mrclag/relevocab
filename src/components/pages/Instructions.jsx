import React from 'react';

import styled from 'styled-components';

const Instructions = () => {
  return (
    <InstructionsWrapper>
      <h1>Get Started:</h1>
      <div style={{ width: '40vw' }}>
        <div className="instructions">
          <ul>
            <li>Add a new deck</li>
            <li>
              Add new cards to your deck on the bottom of the sidebar, or select
              Find Words to search vocabulary relevant to the search term.
            </li>
            <li>Select a deck to begin practicing.</li>
            <li>
              Press Good to remove card from pile, and Again to place the card
              back in the deck.
            </li>
          </ul>
          <br />
        </div>
      </div>
    </InstructionsWrapper>
  );
};

export default Instructions;

const InstructionsWrapper = styled.div`
  margin-left: 400px;
  margin-top: 20vh;
  font-size: 1.25em;
  li {
    margin: 30px 0;
  }
`;
