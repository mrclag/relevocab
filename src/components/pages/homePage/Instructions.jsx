import React from 'react';

import { InstructionsWrapper } from './Instructions.styles';

export const Instructions = () => {
  return (
    <InstructionsWrapper>
      <div className="welcome-wrapper">
        <div className='welcome-title'>Welcome to Vocarta!</div>
        <div>
          To get started,
          <ol>
            <li>
              Add new deck on sidebar, or find premade decks on Songs page.
            </li>
            <li>
              Add new cards to your deck, or select Find Words to search for and
              add related vocabulary.
            </li>
            <li>Select a deck to begin practicing.</li>
            <li>
              Press Good to remove card from pile, and Again to place card back
              in deck.
            </li>
          </ol>
          <br />
        </div>
      </div>
    </InstructionsWrapper>
  );
};

export default Instructions;
