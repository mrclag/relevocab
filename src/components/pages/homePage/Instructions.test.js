import React from 'react';
import { Instructions } from './Instructions.jsx';
import { shallow } from 'enzyme';

describe('Instructions', () => {
  const instructions = shallow(<Instructions />);

  it('renders correctly', () => {
    expect(instructions).toMatchSnapshot();
  });
});
