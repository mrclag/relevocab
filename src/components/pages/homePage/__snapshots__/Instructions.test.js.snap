// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions renders correctly 1`] = `ShallowWrapper {}`;
